ops/helmfiles/dagster/helmfile.yaml

```diff
@@ -55,7 +55,6 @@ releases:
             - name: monster-dagster-secrets
       - userDeployments:
           # "user code" we deploy to dagster; these are dagster repositories of pipelines
-          enabled: true
```

Comment:
(not really part of the downgrade, but this line is redundant with default values)